### DISCUSSION TITLE
Is there a way for uploaded files to be stored on the file system instead of being saved in the database?

### DISCUSSION THREAD

**Discussion by niknam-mohsen** [requarks/wiki#7068] [Help / Questions] [answered]
When I create a new page and want to upload a new image from insert assets, after uploading this image, it will be stored in the database and a corresponding entry will be created in the `dbo.assetData` and `dbo.assets` table.
I want all files to be stored in the local file system on disk. Is there any way?

**Comment by NGPixel** [comment, marked as answer]
No

**Comment by niknam-mohsen** [comment]
> No

Isn't there a solution planned for version 3?

**Reply by NGPixel** [reply]
Yes, v3 will offer that function. This is however not possible in v2.

**Reply by niknam-mohsen** [reply]
When will version 3 be released?
